railties/lib/rails/generators/rails/app/templates/config/initializers/content_security_policy.rb.tt

```diff
@@ -0,0 +1,20 @@
+# Define an application-wide content security policy
+# For further information see the following documentation
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+
+Rails.application.config.content_security_policy do |p|
```

Comment:
Should this config be commented out for upgrading apps? Or should it be loose enough that apps can upgrade seamlessly and get more security?

Reply:
Do initializers get generated for upgrading apps automatically?

Reply:
IIRC, yes.

Reply:
Okay, will investigate what the preferred solution is to be once the beta ships and it's easier to test against upgrading real world apps.